components/catalogs/front/src/components/ProductMapping/components/TargetSourceAssociation/TargetSourceAssociation.tsx

```diff
@@ -41,19 +42,25 @@ export const TargetSourceAssociation: FC<Props> = memo(
                     {targetLabel ?? targetCode}
                     {isRequired && <RequiredPill level='warning' data-testid='required-pill' />}
                 </TargetCell>
-                {(null === source || null === source.source) && (
+                {null === source.source && (undefined === source.default || null === source.default) && (
```

Comment:
```suggestion
                {null === source.source && undefined === source.default && (
```
since empty values are removed from the source so it can only be undefined or am I wrong?